src/App.tsx

```diff
@@ -45,6 +45,11 @@ const App: React.FC = () => {
                 <Route path="/" exact component={Home} />
                 <Route path="/block-template" exact component={BlockTemplate} />
                 <Route path="/grid-template" exact component={GridTemplate} />
+                <Route
+                  path="/signup-directory"
+                  exact
+                  component={SignupDirectory}
```

Comment:
Route is temporary for testing, planned to be compressed into a signup route along with the other signup components